### ISSUE TITLE
Read timeout values from http.Server

### ISSUE THREAD

**Issue by lucas-clemente** [lucas-clemente/quic-go#410]
https://tip.golang.org/pkg/net/http/#Server

**Comment by ohoushyar** [comment]
Hi there, just wanted to reach out and see if the ticket as the status said still needs help?

**Comment by ohoushyar** [comment]
Hey there, I updated server side to read timeout from http.Server. I was also thinking to create a new ticket(s) to address server and client separately, but then I realized that client side implementation had been completed after #2091 (maybe I'm wrong though). If my assumption is right, then I can proceed and send a PR without new ticket. Any objection/comment?

**Comment by marten-seemann** [comment]
@ohoushyar Sure, always happy about PRs :)
